### PR TITLE
Underscores is an ad lib script with awful responses.

### PR DESCRIPTION
**Pull request by drobati**
# Underscores
Planning on populating the database to use [cards against humanity answers](https://github.com/nodanaonlyzuul/against-humanity/blob/master/answers.txt).

The bot should respond to messages that use repeating underscores, by repeating the message back with the underscores replaced by a entry from the database.

I may prune the data to remove all those underscores, and lowercase everything.

### Example
```
<devhost> Sometimes I eat _____.
<hayt> Sometimes I eat poor people.
```

Thoughts welcomed. :hammer:

### CODE REVIEW

**Comment by drobati** [comment]
Done. Please review and merge.

**Comment by sshirokov** [comment]
At the very least move `answers.txt` into a directory other than root.

**Comment by drobati** [comment]
@sshirokov Fixed.

File: scripts/underscores.coffee [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+fs = require('fs')
+
+loadBrain = (robot) ->
+  fs.readFile 'data/answers.txt', 'utf8', (err, contents) =>
+    if err then throw err
+    list = contents.toString().split "\n"
+    robot.brain.set('adlib', list)
+
+module.exports = (robot) ->
+  robot.hear /(_{3,})+/g, (res) ->
```
Comment (sshirokov) [inline scripts/underscores.coffee]: I think by convention these params are `msg`

Comment (drobati) [inline scripts/underscores.coffee]: Are you referring to the params passed at line 21?

Comment (drobati) [inline scripts/underscores.coffee]: s/contents/msg/?

Comment (sshirokov) [inline scripts/underscores.coffee]: No one cares.

**Pull request merged**